A14-channelz.md

```diff
@@ -701,16 +701,18 @@ message Subchannel {
   repeated SocketRef socket = 5;
 }
 
+enum ChannelConnectivityState {
```

Comment:
Enums function oddly outside of a message.  If you want to move this, consider wrapping it in a separate message.

The issue is that enums are not namespaced in Go or CPP, so you end up with top level variables like UNKNOWN, but it isn't clear its a ChannelConnectivityState

Reply:
Done